unified_planning/io/pddl_writer.py

```diff
@@ -686,7 +686,20 @@ def _write_problem(self, out: IO[str]):
             else:
                 out.write(f" (= {converter.convert(f)} {converter.convert(v)})")
         if self.problem.kind.has_actions_cost() or self.problem.kind.has_plan_length():
-            out.write(f" (= (total-cost) 0)")
+            out.write(" (= (total-cost) 0)")
+        for tm, le in self.problem.timed_effects.items():
+            for e in le:
+                out.write(f" (at {tm.delay}")
```

Comment:
If not an integer would output a fraction.

For instance, round trip reading/writing of the truck domain gives me : `(at 9197/10 (not (deliverable package1 l1)))`

Reply:
You are right!